tsfile/src/main/java/org/apache/iotdb/tsfile/compress/IUnCompressor.java

```diff
@@ -312,4 +315,45 @@ public CompressionType getCodecName() {
       return CompressionType.GZIP;
     }
   }
+
+  class ZstdUnCompressor implements IUnCompressor {
+
+    @Override
+    public int getUncompressedLength(byte[] array, int offset, int length) throws IOException {
+      return (int) Zstd.decompressedSize(array, offset, length);
```

Comment:
Is this a time-consuming call?

Reply:
It looks like the size is saved in the header.
![image](https://user-images.githubusercontent.com/55970239/223661109-c1f64cc2-55d7-4563-8a95-7950c5231892.png)
![image](https://user-images.githubusercontent.com/55970239/223661110-247c64dd-4781-4600-a521-2512aedd12d5.png)